### ISSUE TITLE
Unable to reproduce the Multinex LOL-v1 training performance using the official configuration

### ISSUE THREAD

**Issue by Xiaoggg221** [albrateanu/multinex#7]
Hi authors,

Thank you for releasing the code and pretrained models for Multinex.

I successfully evaluated the official pretrained checkpoint on the LOL-v1 test set and obtained the following results:

- PSNR: 23.1828 dB
- SSIM: 0.8426
- LPIPS: 0.1248

These results are consistent with the performance reported in the paper, which suggests that my LOL-v1 test dataset and evaluation pipeline are likely correct.

However, when I retrained the model from scratch using the released configuration, I was unable to reproduce the reported performance.

My training command was:

```bash
python -m basicsr.train --opt Options/Multinex_LOL-v1.yaml
```
The best checkpoint obtained during training is shown below:
<img width="381" height="170" alt="Image" src="https://github.com/user-attachments/assets/5261e0fc-7303-4508-a3cc-27e01c403886" />
